string module_name
uint16 error_code
